### ISSUE TITLE
Option to remove "charset=UTF-8" from Content-Type header

### ISSUE THREAD

**Issue by Cyb0rk** [Kong/unirest-java#305]
Currently I'm using 
```
fileUploadResponse = Unirest.post("https://httpbin.org/post")
  .field("key", "keyname")
  .field("name", "Testname")
  .field("type", "testtype")
  .field("file", file, "Testfile.testtype")
  .asString();
```

to upload a file as a POST request.

Because it adds "charset=UTF-8" in the Content-Type header the request could not be processed. So I have to use and very old version in gradle : `"group: 'com.mashape.unirest', name: 'unirest-java', version: '1.4.9'"` to fix this issue.

Is it possible to add an option to remove the encoding parameter in Content-Type header?

**Comment by ryber** [comment]
So httpbin, as well as my own spark test server are totally ok with this and process the request just fine. So I assume you have some other particular server that has a problem with it. I re-read the RFC for this section and in somewhat typical http fashion it's a bit vague

You can remove the overall encoding for the entire multipart section by setting the content to null like so:

```
HttpResponse<String> r = Unirest.post("https://httpbin.org/post")
                .field("key", "keyname")
                .field("name", "Testname")
                .field("type", "testtype")
                .field("file", rezFile("/test"), "Testfile.testtype")
                .charset(null)
                .asString();
```

**Comment by ryber** [comment]
@Cyb0rk did this answer work for you?

**Comment by Cyb0rk** [comment]
@ryber Works fine, sorry that I didn't figure it out by myself. Thank you very much!

**Comment by Serhiy-iLchenko** [comment]
Just had the same issue with PHP-based backed not pulling attachment into $_FILES unless I remove charset from Content-type.
Please add something more obvious instead of null, like .noCharset()

**Comment by ryber** [comment]
adding a little sugar for this to make it more obvious

**Comment by ryber** [comment]
released in 3.11.00
